lib/heuristics/concerns/scheduling_data_initialisation.rb

```diff
@@ -311,4 +309,22 @@ def compute_capacities(quantities, vehicle, available_units = [])
 
     capacities
   end
+
+  def exist_common_multiplexer?(lapses)
```

Comment:
[multiplexer](https://en.wikipedia.org/wiki/Multiplexer) has a very specific meaning on electronics :thinking:  do we mean "multiple", "divisor" something like that here?

If we also name the function `has_common_X?` or `common_X_exists?`, it would be less Yodish, what do you think?

Reply:
I didn't mark every appearance of the word `multiplexer` but can you replace them with the correct word as well?

Reply:
After reading the function in detail, I think the word you are looking for is "divisor" but not sure.